hie-plugin-api/Haskell/Ide/Engine/Ghc.hs

```diff
@@ -1,3 +1,4 @@
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
```

Comment:
Ditto for all the others

Reply:
I think this is already covered by us setting `-Wall` in the cabal file?

Reply:
To clarify, I believe it is a good thing to warn about unused imports, and this should happen. Throughout the project.